.github/workflows/profile-load-test.yml

```diff
@@ -0,0 +1,53 @@
+# description: This workflow allows to profile more easily running load tests
+# type: CI
+# owner: @camunda/zeebe-distributed-platform
+name: Profile running load test
+on:
+  workflow_dispatch:
+    inputs:
+      name:
+        description: 'Specifies the name of the load test to profile'
+        required: true
+      pod:
+        description: 'Specifies the pod name of the load test to profile. Please only specify the suffix like zeebe-0, or zeebe-1, etc.'
+        default: 'zeebe-0'
+        required: false
+jobs:
+  profile-load-test:
+    name: Async profiling running load test
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    permissions:
+      contents: 'read'
+      id-token: 'write'
```

Comment:
👀 Why write permissions for `id-token`?

Reply:
Hm this was copied from the benchmark workflow. I can try to remove it 🤔

Reply:
https://stackoverflow.com/questions/72183048/what-is-the-permission-scope-of-id-token-in-github-action

It is expected for the OIDC thingy